physical/zookeeper.go

```diff
@@ -162,28 +163,29 @@ func (c *ZookeeperBackend) ensurePath(path string, value []byte) error {
 // cleanupLogicalPath is used to remove all empty nodes, begining with deepest one,
 // aborting on first non-empty one, up to top-level node.
 func (c *ZookeeperBackend) cleanupLogicalPath(path string) error {
-	nodes := strings.Split(path, "/")
+	pathSeparator := fmt.Sprintf("%c", os.PathSeparator)
```

Comment:
This and 1969 are basically the same, except that this is actually not correct -- zookeeper isn't a local filesystem so doesn't use local filesystem path separators -- like Consul it's always '/'.